### DISCUSSION TITLE
[Question] Best way to get a copy of work in progress before a release

### DISCUSSION THREAD

**Discussion by keshmir** [fomantic/Fomantic-UI#688] [General] [answered]
Hi all,

Sorry if this question is obvious, I am new to github.

There are times that an issue is fixed and I want to get a copy of a particular patch and verify it against my site.  There are also times that there is a milestone in progress but not released yet.  Can someone give me basic steps on the proper method to get a snapshot of the code for either scenario?

**Comment by ko2in** [comment]
@keshmir , I think you're asking to get the source for certain commit. Maybe this [post](https://stackoverflow.com/questions/7832770/how-to-get-certain-commit-from-github-project) on SO will give you some hints.

**Comment by lubber-de** [comment, marked as answer]
For merged PR's which are not yet released as a version, you can checkout the develop branch and build or use the files from there.
What I am usually doing for Testcase is including separate files directly from my branches
https://gomakethings.com/how-to-turn-any-github-repo-into-a-cdn/

**Comment by lubber-de** [comment]
@keshmir is your question answered so far? Can we close this?

**Comment by lubber-de** [comment]
Assuming, it's answered thus closing.
